### PR TITLE
Export validator

### LINKED ISSUE #443: Implement export validator keystore and slashing db

### PR DESCRIPTION
**Pull request by BeroBurny**
closes #443

### CODE REVIEW

File: src/renderer/ducks/validator/sagas.ts [hunk 1]
```diff
@@ -342,6 +346,55 @@ function* onBeaconNodeStatusChangeUpdateValidatorState(
...
+                controller: cgDbController,
+            });
+            const genesis = yield call(new CgEth2ApiClient(config, validator.beaconNodes[0]).beacon.getGenesis);
+            db = yield call(slashingProtection.exportSlashingJson, genesis.genesisValidatorsRoot, meta);
+        }
+
+        const name = validator.name
+            .toLowerCase()
+            .replace(/ /g, "_")
+            .replace(/[^a-zA-Z0-9_]/g, "");
```
Comment (mpetrunic) [inline src/renderer/ducks/validator/sagas.ts:375]: I think we are already doing something similar somewhere. Extract to utility method?

File: src/renderer/services/eth2/client/slashingProtection/index.ts [hunk 2]
```diff
@@ -11,4 +13,16 @@ export class CGSlashingProtection extends SlashingProtection {
 
         return !(signedBlocks.length || signedAttestations.length);
     };
+
+    public exportSlashingJson = (genesisValidatorsRoot: Root, publicKey: string): Promise<Interchange> => {
+        const uintPublicKey = new Uint8Array(Buffer.from(publicKey.substr(2), "hex"));
```
Comment (mpetrunic) [inline src/renderer/services/eth2/client/slashingProtection/index.ts:18]:
```suggestion
        const uintPublicKey = fromHex(publicKey)
```

File: src/renderer/services/utils/account.ts [hunk 3]
```diff
@@ -78,29 +78,20 @@ export const deleteBeaconNodeContainers = async (): Promise<void> => {
...
-        const copyResult = copyFile(keystorePath, savePath);
-        return {
-            level: copyResult.success ? Level.INFO : Level.ERROR,
-            message: copyResult.success
-                ? `Successfully exported keystore for validator ${validator.name} to ${savePath}.`
-                : `Export failed: ${copyResult.message}.`,
-        };
+    // store slashing DB
+    if (slashingDB) {
+        const slashingDBSavePath = path.join(exportPath, `${name}-slashing-db.json`);
```
Comment (mpetrunic) [inline src/renderer/services/utils/account.ts:94]: putting date and time into slashing db file might be good idea

Comment (BeroBurny) [reply src/renderer/services/utils/account.ts:94]: export both files in same time, we need timestamp for it?

File: src/renderer/ducks/validator/sagas.ts (hunk 1 above)
Comment (BeroBurny) [reply src/renderer/ducks/validator/sagas.ts:375]: its similar not the same

File: src/renderer/services/utils/account.ts (hunk 3 above)
Comment (mpetrunic) [reply src/renderer/services/utils/account.ts:94]: I meant just adding timestamp to this file. Timestamp is irrelevant for keystore but quite relevant for slashing db

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**